scripts/format-codes.sh

```diff
@@ -21,7 +21,7 @@ FOLDERS_TO_FORMAT=("dags" "xlml")
 
 for folder in "${FOLDERS_TO_FORMAT[@]}"
 do
-  pyink "$folder" --pyink-indentation=2 --pyink-use-majority-quotes
+  ~/py39/bin/python3 -m  pyink "$folder" --pyink-indentation=2 --pyink-use-majority-quotes
```

Comment:
Why didn't the old version work, does the PATH not include the installed Python binaries?

Also, we should avoid linking to a specific Python version if possible.

Reply:
oops, I need to revert this. On my local machine I needed to create a pyenv to install this package.